src/test/regress/expected/multi_table_ddl.out

```diff
@@ -88,6 +88,39 @@ SELECT 1 FROM master_add_node('localhost', :worker_2_port);
         1
 (1 row)
 
+-- test recovery when removing node
+CREATE TABLE recovery_test (x int, y int);
+SELECT create_distributed_table('recovery_test','x');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+DROP TABLE recovery_test;
+SELECT master_remove_node('localhost', :worker_2_port);
```

Comment:
can we move this test to a more relevant file, like `multi_remove_node_reference_table.sql`.

(I know that I suggested it should be OK to have this test anywhere, but the newly suggested is slightly more appropriate. )